grammars/sql.cson

```diff
@@ -181,6 +181,10 @@
     'name': 'keyword.other.case.sql'
   }
   {
+    'match': '(?i)\\b(coalesce|nvl|nullif)\\b'
+    'name': 'keyword.other.conditional_expression.sql'
```

Comment:
Thank you for reviewing. Sounds good and I updated https://github.com/atom/language-sql/pull/67/commits/c48410f9201ff2619b8feead5fd743a6ed55423c